libmscore/text.cpp

```diff
@@ -198,8 +198,14 @@ bool TextFragment::operator ==(const TextFragment& f) const
 
 void TextFragment::draw(QPainter* p, const Text* t) const
       {
-      p->setFont(font(t));
```

Comment:
I think I will add a #ifdef Q_OS_WIN here. It seems a pity to do all the following if DPIFACTOR is 1.0. Otherwise, I don't see any artefact on MacOS. So I'll merge this so we can get more testers.